lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -103,11 +103,17 @@
             (select
                 (setq (? set_quantifier))
                 (project projection)
+                (project_let (? let))
                 (from from_source)
+                (from_let (? let))
                 (where (? expr))
+                (where_let (? let))
                 (group (? group_by))
+                (group_let (? let))
                 (having (? expr))
-                (limit (? expr))))
+                (having_let (? let))
+                (limit (? expr))
+                (limit_let (? let))))
```

Comment:
This is a reasonable way to model this now, just add a comment referencing [partiql-ir-generator #44](https://github.com/partiql/partiql-ir-generator/issues/44).

When that issue is completed I'd like to model this with a variadic record element instead, i.e.

```
(select
    ... 
    (project_let (* let_binding 0))
    ...)
```

Reply:
Added.